### DISCUSSION TITLE
How to add screenshot to distinct step in gauge report

### DISCUSSION THREAD

**Discussion by flexin777** [getgauge/gauge#1938] [General]
Hello
Is it possible to add screenshot to step? 
I am using Gauge+Taiko+js. And I want to make a screenshot and add it to gauge report. How can I do it?

**Comment by zabil** [comment]
This is documented here
https://docs.taiko.dev/taking_screenshots/#taking-screenshots-with-gauge
and here
https://docs.gauge.org/writing-specifications.html?os=linux&language=javascript&ide=vscode#adding-screenshots-in-reports

**Comment by flexin777** [comment]
Thank you!

вт, 16 мар. 2021 г. в 16:52, Zabil Cheriya Maliackal <
***@***.***>:
